docs/nx/next/react-query.md

```diff
@@ -0,0 +1,81 @@
+<!-- markdownlint-disable MD041 -->
+### @ensono-stacks/next:react-query
+
+<details>
+<summary>Add Next Authentication to your next application</summary>
+
+The react-query generator will install and update `_app.tsx` into an existing Next application. It will update the `_app.tsx` file by adding the query provider and update the `.eslintrc` file with the react-query eslint plugin.
+
+
+## Prerequisites
+
+An existing [Next](https://nextjs.org/) application
```

Comment:
Is _app.tsx always present?

Reply:
If they are using pages it will always have `_app.tsx`